src/main/java/org/broadinstitute/hellbender/engine/AlignmentContext.java

```diff
@@ -122,10 +122,16 @@ public AlignmentContext stratify(final ReadOrientation type) {
      * Splits the given AlignmentContext into a StratifiedAlignmentContext per sample, but referencd by sample name instead
      * of sample object.
      *
+     * @param assumedSingleSample assumed single sample in the file or null if there is none (eg file contains more than 1 sample).
```

Comment:
Wording is a little confusing here -- how about: `If non-null, assume this is the only sample in our pileup and return immediately. If null, get the list of samples from the provided header.`